system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -148,7 +186,7 @@
                     {:accept-format :json
                      :raw? true})
           {:keys [errors]} (ingest/parse-ingest-body :json response)]
-      (is (re-find #"Request content is too short." (first errors)))))
+      (is (re-find #"required key \[Name\] not found" (first errors)))))
```

Comment:
it started out capitalized before and now it is not. If this is intended then ignore me.